app/views/locations/index.html.erb

```diff
@@ -0,0 +1,110 @@
+<div class="row">
+  <div class="large-4 small-12 columns">
+
+    <img src="http://placehold.it/500x500&text=Logo">
+
+    <div class="hide-for-small panel">
+      <h3>Header</h3>
+      <h5 class="subheader">Risus ligula, aliquam nec fermentum vitae, sollicitudin eget urna. Donec dignissim nibh fermentum odio ornare sagittis.
+      </h5>
+    </div>
+
+    <a href="#">
+    <div class="panel callout radius">
+      <h6>99  items in your cart</h6>
+    </div>
+    </a>
```

Comment:
You are correct! We are going to end up removing this and further modifying the template, however we just wanted to get the overall template up so that both pairs are working from the same general layout. Perhaps I should have labeled WiP?